lng/english.lng.php

```diff
@@ -2066,3 +2066,6 @@
 $lng['serversettings']['letsencryptecc']['description'] = "If set to a valid key-size the certificate issued will use ECC / ECDSA";
 $lng['serversettings']['froxloraliases']['title'] = "Domain aliases for froxlor vhost";
 $lng['serversettings']['froxloraliases']['description'] = "Comma separated list of domains to add as server alias to the froxlor vhost";
+$lng['dkim']['key_under_construction'] = "Keys are under construction. Please wait approx. 5 minutes...";
```

Comment:
remove timespecification as you don't know the system. First sentence is enough

Reply:
done.